pkg/agent/baker.go

```diff
@@ -480,7 +480,7 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			return profile.Distro.IsKataDistro()
 		},
 		"IsCustomImage": func() bool {
```

Comment:
can we add something in baker_test.go to make sure this piece of the func map gets called correctly?

basically a test where the node bootstrapping config specifies a custom kata image

Reply:
Good point! Added a test case.